test/pgtest-mz/ddl-extended.pt

```diff
@@ -0,0 +1,49 @@
+# Test that multiple DDL statements in the same extended request are supported
```

Comment:
How does this differ from postgres?

Reply:
In Postgres the Insert will be aborted while in Materialize the Insert will be committed in it's own transaction. I've added a `SELECT` at the end to clarify this.